### PR TITLE
[bitnami/nginx] add `cloneStaticSiteFromGit.extraEnvVarsSecret`

### PR DESCRIPTION
**Pull request by smehboub**
### Description of the change

 Add `cloneStaticSiteFromGit.extraEnvVarsSecret` value

### Benefits

Allow to use credentials as secret mounted as variables to clone the content of a private repository by using a token instead of a ssh key.
With this, we can use cloneStaticSiteFromGit.repository: "https://oauth2:${token}@github.com/example/test.git" by pickup the token from a secret.

### Possible drawbacks

None

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by smehboub** [comment]
Hello @andresbono,

Kindly please, is it possible to have any news about this MR ?

Thanks in advance.
Rgs,

File: bitnami/nginx/values.yaml [hunk 1]
```diff
@@ -461,6 +461,9 @@ cloneStaticSiteFromGit:
     ## @param cloneStaticSiteFromGit.gitClone.args Override default container args for git-clone-repository
     ##
     args: []
+    ## @param cloneStaticSiteFromGit.gitClone.extraEnvVarsSecret Secret with extra environment variables
+    ##
+    extraEnvVarsSecret: ""
```
Comment (andresbono) [inline bitnami/nginx/values.yaml:466]: For the use case that you described, it seems the token would be only necessary in the `git-clone-repository` initContainer, right? I guess that is why you included the new value under `gitClone`.

But I wonder if it wouldn't be more convenient to generalize `extraEnvVarsSecret` and set it under `cloneStaticSiteFromGit.extraEnvVarsSecret`, adding it for the 2 containers: `git-clone-repository` and `git-repo-syncer`. That would give the value more flexibility for different use cases that other users may come up with.

What do you think?

**Review by andresbono — COMMENTED** [review]
Thank you so much for your contribution! Could you please check my comments?

**Comment by smehboub** [comment]
> Thank you so much for your contribution! Could you please check my comments?

Thank you for your return!
I commited your proposition :-)

File: bitnami/nginx/values.yaml (hunk 1 above)
Comment (smehboub) [reply bitnami/nginx/values.yaml:466]: Yes it's a good idea, for the value flexibility for different use case.
I will make the change...

Comment (smehboub) [reply bitnami/nginx/values.yaml]: The change is done ;-)

**Review by andresbono — APPROVED** [review]
LGTM thank you very much!

I made a minor change (moving the value below `extraEnvVars`).

**Pull request merged**